homeassistant/components/fritzbox/config_flow.py

```diff
@@ -36,30 +41,31 @@
 RESULT_SUCCESS = "success"
 
 
-class FritzboxConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+class FritzboxConfigFlow(ConfigFlow, domain=DOMAIN):
     """Handle a AVM FRITZ!SmartHome config flow."""
 
     VERSION = 1
 
-    def __init__(self):
+    def __init__(self) -> None:
         """Initialize flow."""
-        self._entry = None
-        self._host = None
-        self._name = None
-        self._password = None
-        self._username = None
+        self._entry: ConfigEntry | None = None
+        self._host: str | None = None
+        self._name: str = ""
+        self._password: str | None = None
+        self._username: str | None = None
 
-    def _get_entry(self):
+    def _get_entry(self, name: str) -> FlowResult:
         return self.async_create_entry(
-            title=self._name,
+            title=name,
```

Comment:
```suggestion
            title=str(name),
```

@KapJI  Would this be ok to do?

Reply:
If `name` argument is already a `str` this shouldn't be needed.